src/trimgalore/config.vsh.yaml

```diff
@@ -0,0 +1,314 @@
+        required: false
+      - name: --non_directional
+        type: boolean
+        description: Selecting this option for non-directional RRBS libraries will screen quality-trimmed sequences for 'CAA' or 'CGA' at the start of the read and, if found, removes the first two basepairs. Like with the option '--rrbs' this avoids using cytosine positions that were filled-in during the end-repair step. '--non_directional' requires '--rrbs' to be specified as well. Note that this option does not set '--clip_r2 2' in paired-end mode.
+        required: false
+      - name: --keep 
+        type: boolean
+        description: Keep the quality trimmed intermediate file. 
+        required: false
+  - name: Paired-end specific options
+    arguments: 
+      - name: --paired 
+        type: boolean
+        description: This option performs length trimming of quality/adapter/RRBS trimmed reads for paired-end files. To pass the validation test, both sequences of a sequence pair are required to have a certain minimum length which is governed by the option --length (see above). If only one read passes this length threshold the other read can be rescued (see option --retain_unpaired). Using this option lets you discard too short read pairs without disturbing the sequence-by-sequence order of FastQ files which is required by many aligners. Trim Galore expects paired-end files to be supplied in a pairwise fashion, e.g. file1_1.fq file1_2.fq SRR2_1.fq.gz SRR2_2.fq.gz ... .
+        required: false
+      - name: --retain_unpaired  
+        type: boolean
+        description: If only one of the two paired-end reads became too short, the longer read will be written to either '.unpaired_1.fq' or '.unpaired_2.fq' output files. The length cutoff for unpaired single-end reads is governed by the parameters -r1/--length_1 and -r2/--length_2. 
+        required: false
+      - name: --length_1
+        alternatives: -r1
+        type: integer 
+        description: Unpaired single-end read length cutoff needed for read 1 to be written to '.unpaired_1.fq' output file. These reads may be mapped in single-end mode.
+        example: 35 
+        required: false
+      - name: --length_2
+        alternatives: -r2
+        type: integer 
+        description: Unpaired single-end read length cutoff needed for read 2 to be written to '.unpaired_2.fq' output file. These reads may be mapped in single-end mode.
+        required: false
+        example: 35   
+  - name: Output
+    arguments:
+      - name: --output_dir
+        alternatives: -o
+        type: file
+        description: If specified all output will be written to this directory instead of the current directory. 
+        direction: output
+        required: false
```

Comment:
It strikes me that none of the outputs are mandatory. What happens if you specify none?

Reply:
At the moment, a default output directory is being created.